src/ui/common/src/components/integrations/cards/emailCard.tsx

```diff
@@ -0,0 +1,38 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+
+import { EmailConfig, Integration } from '../../../utils/integrations';
+
+type Props = {
+  integration: Integration;
+};
+
+export const EmailCard: React.FC<Props> = ({ integration }) => {
+  const config = integration.config as EmailConfig;
+  const target = (JSON.parse(config.targets_serialized) as string[])[0];
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
This information presentation is a little tough to parse. @agiron123, any ideas on how we can make this cleaner?

Reply:
We could put only the sender email address and make everything else in a drop-down

Reply:
@vsreekanti Something easy that we can do here is group together related elements. Host and port will be one group, and sender address and receiver address will be another group then level will be it's own group.

Personally I think we should put level at the top and then show the config information. It'd be nice to use some iconography here as well to show the succeeded/warning/error icon next to the text for the notification level.

Ultimately there's two groups, one for Notification level, and one for Configuration Information. You can make a header for the two groups and indent the text below to show some hierarchy. I'd like to also explore common patterns for our other Notification related integrations to see if there are common themes so we can have common page headers in each integration.